client/src/components/AuthModal/SignInTabPanel.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { FC } from 'react'
+import FormGroup from '@material-ui/core/FormGroup'
+import FormControl from '@material-ui/core/FormControl'
+import TextField from '@material-ui/core/TextField'
+import Button from '@material-ui/core/Button'
+import { useFormik } from 'formik'
+import * as yup from 'yup'
+import { useActions } from '../../hooks/useActions'
+
+const validationSchema = yup.object({
```

Comment:
validation schemas should be placed in different files, it would increase the readability of your code

Reply:
fix